packages/helpers/classes/personalization.d.ts

```diff
@@ -7,7 +7,7 @@ export interface PersonalizationData {
   subject?: string;
   headers?: { [key: string]: string };
   substitutions?: { [key: string]: string };
-  dynamicTemplateData?: { [key: string]: string; };
+  dynamicTemplateData?: { [key: string]: any; };
```

Comment:
This works, but there are other areas we need to change, see [this PR](https://github.com/sendgrid/sendgrid-nodejs/pull/868) + [line 115](https://github.com/sendgrid/sendgrid-nodejs/blob/ab7dc11bf3ad597321ecbdd59f1480998d3f492c/packages/helpers/classes/personalization.d.ts#L115)

Reply:
true, let me fix it as well

Reply:
How about here? https://github.com/sendgrid/sendgrid-nodejs/blob/ab7dc11bf3ad597321ecbdd59f1480998d3f492c/packages/helpers/classes/personalization.d.ts#L115

Reply:
i've fixed it next commit 👇